gradle.properties

```diff
@@ -4,8 +4,8 @@ kotlin.code.style=official
 kotlin.mpp.stability.nowarn=true
 android.useAndroidX=true
 
-kaluga.androidGradlePluginVersion=7.0.1
-kaluga.kotlinVersion=1.5.21
+kaluga.androidGradlePluginVersion=7.0.4
```

Comment:
I think you had problem upgrading this to latest stable? (7.1.0) If so, could you make an issue with what problems you had?